app/src/main/res/values/strings.xml

```diff
@@ -965,9 +965,10 @@
     <string name="onboarding_whats_new_description">Have questions about the redesigned %s? Want to know what’s changed?</string>
     <!-- text for underlined clickable link that is part of "what's new" onboarding card description that links to an FAQ -->
     <string name="onboarding_whats_new_description_linktext">Get answers here</string>
-    <!-- text for the firefox account onboarding card header
-    The first parameter is the name of the app (e.g. Firefox Preview) -->
-    <string name="onboarding_firefox_account_header">Get the most out of %s.</string>
+    <!-- text for the firefox account onboarding card header -->
+    <string name="onboarding_firefox_account_header">Start syncing bookmarks, passwords, and more with your Firefox account.</string>
```

Comment:
We need to create a new ID for this new string or weirdness happens (see https://github.com/mozilla-mobile/fenix/issues/14129)

Reply:
cc @kglazko: when we do string changes, we need to change the id - you can't just rename the string contents.

For now, we don't want to delete the old id either because it causes problems when uplifting strings from the L10N import bot, so just create a new one.